README.md

```diff
@@ -148,7 +150,12 @@ For the Common Custom User Data Maven extension which is defined with the `ccudM
 To authenticate against the Develocity server, you should specify a masked environment variable named `DEVELOCITY_ACCESS_KEY`.
 See [here](https://docs.gitlab.com/ee/ci/variables/#define-a-cicd-variable-in-the-ui) on how to do this in GitLab UI.
 To generate a Develocity Access Key, you can check [Develocity Gradle plugin docs](https://docs.gradle.com/enterprise/gradle-plugin/#manual_access_key_configuration) and [Develocity Maven extension docs](https://docs.gradle.com/enterprise/maven-extension/#manual_access_key_configuration).
-A short-lived access token will be retrieved given the access key and will replace the `DEVELOCITY_ACCESS_KEY` variable.
+
+### Short-lived access tokens
+A short-lived access token will be retrieved given the access key and will replace the `DEVELOCITY_ACCESS_KEY` variable content.
+If a short-lived token fails to be retrieved (for example the Develocity server version is lower than 2024.1), no access key will be set.
```

Comment:
What happens if no access key will be set? Please add this to the edits I did below.